data-prepper-core/src/main/java/org/opensearch/dataprepper/plugin/PluginConfigurationConverter.java

```diff
@@ -31,6 +31,10 @@ class PluginConfigurationConverter {
     PluginConfigurationConverter(final Validator validator,
                                  @Named("pluginConfigObjectMapper")
                                  final ObjectMapper objectMapper) {
+        final SimpleModule simpleModule = new SimpleModule();
+        simpleModule.addDeserializer(Duration.class, new DataPrepperDurationDeserializer()
```

Comment:
This line needs to end with an extra parenthesis. The GitHub Actions are failing with compiler errors.